src/Microsoft.Framework.Runtime.Hosting/RuntimeHostBuilder.cs

```diff
@@ -0,0 +1,28 @@
+﻿using NuGet.ProjectModel;
+
+namespace Microsoft.Framework.Runtime.Hosting
+{
+    public class RuntimeHostBuilder
+    {
+        public string ApplicationBaseDirectory { get; }
+
+        public RuntimeHostBuilder(string applicationBaseDirectory)
+        {
+            ApplicationBaseDirectory = applicationBaseDirectory;
+        }
+
+        public RuntimeHost Build()
+        {
+            // Load the Project
+            Project project = null;
+
+            PackageSpec packageSpec;
+            if (JsonPackageSpecReader.TryReadPackageSpec(ApplicationBaseDirectory, out packageSpec))
```

Comment:
But seriously, it'll throw and we'll catch it in a top-level try...catch in Program. Said 'try...catch' will write to the console AND rethrow (so that debuggers can grab it if they want it). We can revise this later.